.github/workflows/terraform-apply.yaml

```
@@ -115,10 +115,12 @@ jobs:
           terraform plan -no-color \
           -var-file=configs/github.tfvars \
           -var-file=configs/${{ matrix.repository }}.tfvars \
+          -var="pr_branch_prefix=${{ github.head_ref }}" \
+          -var="pr_body=This is an automated pull request from https://github.com/${{ github.repository }}/pull/${{ github.event.number }} to update centrally managed files." \
           -out ./tf.plan
 
-      - name: Terraform apply (if running from main branch)
-        if: ${{ github.ref_name == 'main' }}
+      - name: Terraform apply (if running on PR)
```

```suggestion
      - name: Terraform apply (if running on non-draft PR)
```

Maybe can make it clearer that it won't run for drafts?